releasenotes/notes/no-dateutil-ca2ca5f5a75fbb22.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    The ``python-dateutil`` library is no longer a dependency of Qiskit. Since
+    Qiskit 2.0.0 nothing in the library wa actively using the library anymore
```

Comment:
A small typo:

```suggestion
    Qiskit 2.0.0 nothing in the library was actively using the library anymore
```

You are also using the word "anymore" a bit too many times in one short paragraph. 😛